### PR TITLE
Support SWR

### PR DESCRIPTION
**Pull request by yuleicul**
**Updated**: Since SWR doesn't provide hooks like `useMutation` in React Query, and it seems very flexible to use `mutate` in SWR, this PR will not provide SWR generated code about `mutations`

* Add new argument `Library` to CLI for pointing out the specific library to generate code (choices: 'swr', 'react-query', default: 'react-query')
* Update tests
* Update README
---
Since SWR doesn't provide hooks like `useMutation` in React Query, and it seems very flexible to use `mutate` in SWR, this PR will not provide SWR generated code about `mutations`

* Add new option `--swr` to command for generating code with SWR hooks
* Update tests
* Update the document

### CODE REVIEW

**Comment by rametta** [comment]
Hello @yuleicul, thanks for the contribution. A few points before we proceed:

1. Can we split everything related to SWR into it's own folder? Ideally we wouldn't have the 2 (RQ & SWR) mixed together in the same files. So it would be great if you can add a directory called `swr` and create files for everything that's needed and leave all the other files untouched if possible.
2. For the CLI options, instead of adding `--swr`, we should have a command to specify either one or the other. So something like `rapini react-query --option1 --option2 ...` or `rapini swr --option1 --option2` - that way it will prevent the user from using rq options with swr and vice versa, so preventing confusion. (And keep the old way of specifying commands the same to be backwards compatible)

So far it's great, thanks, keep it up!

**Comment by yuleicul** [comment]
Hi @rametta , thanks for your review. Your comments are valuable and can make this project robuster. I'll try to refactor it in the next few weeks.

**Comment by yuleicul** [comment]
Hi @rametta, updated, please review. Thank you!

File: src/swr/print.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    "description": "This package was generated by Rapini",
+    "main": "dist/index.js",
+    "types": "dist/index.d.ts",
+    "license": "private",
+    "scripts": {
+      "build": "tsc index.ts --declaration --skipLibCheck --outDir dist"
+    },
+    "peerDependencies": {
+      "axios": "0.27.x",
+      "swr": "^1.3.0"
```
Comment (rametta) [inline src/swr/print.ts:23]: Can it be more generic like `1.x.x` or `1.3.x`? I would rather not hardcode to a specific version like this

**Review by rametta — COMMENTED** [review]
Great work so far! I'm gonna have to run it locally some time this week (or next) to make sure everything is working as expected, so give me a bit more time and I'll get back to this soon

File: src/swr/print.ts (hunk 1 above)
Comment (yuleicul) [reply src/swr/print.ts:23]: [Semver](https://docs.npmjs.com/about-semantic-versioning#using-semantic-versioning-to-specify-update-types-your-package-can-accept) here. `1.x` and  `^1.3.0` are the same.

**Review by rametta — APPROVED** [review]
Amazing work, thank you for the contribution @yuleicul !

**Pull request merged**